packages/common/index.ts

```diff
@@ -250,14 +258,24 @@ export type PhoneNotifBody = {
 // =================== API Route Types ===========================
 // On backend, validated with https://docs.nestjs.com/techniques/validation
 // API route Params and Responses
-export type GetClubResponse = Club[];
-
-export type CreateClubParams = { name: string; rating: number };
-export type CreateClubResponse = Club;
 
 // Office Hours Response Types
 export type GetProfileResponse = User;
 
+export class UpdateProfileParams {
+  @IsBoolean()
+  @IsOptional()
+  desktopNotifsEnabled?: boolean;
```

Comment:
minor thing, can we add a new line here?
```suggestion
  desktopNotifsEnabled?: boolean;
  
```